src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -3244,6 +3244,10 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       Name="SplitResourcesByCulture"
       DependsOnTargets="AssignTargetPaths">
 
+    <PropertyGroup>
+      <RespectAlreadyAssignedItemCulture Condition="'$(RespectAlreadyAssignedItemCulture)' == ''">false</RespectAlreadyAssignedItemCulture>
```

Comment:
Is there a rollout plan for making this `true`?

Reply:
Yes, once this is checked in -> enabling possibility to to use the predefined Culture metada functionality. 
Within the next release I'll enable this behaviour by default (under the change wave 17.12). I planned to file a issue not to forget it (after PR is merged).